internal/pkg/service/objectstorage/cloud/aws/s3.go

```diff
@@ -64,6 +64,11 @@ func (s S3ObjectStorageAdapter) CreateBucket(ctx context.Context, bucket *v1alph
 	return err
 }
 
+// UpdateBucket does nothing as we cannot update an s3 bucket
+func (s S3ObjectStorageAdapter) UpdateBucket(ctx context.Context, bucket *v1alpha1.Bucket) error {
+	return nil
```

Comment:
We need to be able to update storage accounts in Azure. I'm not trying to implement this in AWS because this is not needed now and also well, this will be replaced by crossplane